<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* General Window Appearance */
QMainWindow, #centralWidget {
    background-color: #FDF6EC; /* Cream Background */
}


QTableWidget {
    background-color: #FFFFFF;
    color: #5D5D5D; 
    gridline-color: #F0F0F0;
    border: 1px solid #E0E0E0;
    border-radius: 8px;
    font-family: &quot;Segoe UI&quot;, sans-serif;
    font-size: 13px;
}

QHeaderView::section {
    background-color: #F0F0F0;
    color: #888888;
    padding: 6px;
    border: none;
    font-weight: bold;
    text-transform: uppercase;
}


QPushButton {
    background-color: #A8DADC; 
    color: #FFFFFF;
    border-radius: 12px;
    padding: 8px 15px;
    font-weight: bold;
    border: none;
}

QPushButton:hover {
    background-color: #81B2B4;
}

/
#favButton {
    background-color: transparent;
 	color:#fff
    font-size: 24px;
}

#favButton:hover {
    color: #F1948A;
}

#deleteButton {
    background-color: #F5A9A9; 
    color: white;
}

#deleteButton:hover {
    background-color: #F1948A;
}

/* Mood Sidebar (List Widget) */
QListWidget {
    background-color: #FFFFFF;
    color: #5D5D5D;
    border: 1px solid #E0E0E0;
    border-radius: 8px;
    padding: 5px;
}

QListWidget::item {
    padding: 10px;
    border-radius: 4px;
}

QListWidget::item:selected {
    background-color: #E8F4F8; 
    color: #5D5D5D;
    font-weight: bold;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">centralwidget
{

    background-color: rgb(85, 170, 255);
}
</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_3" rowstretch="1,8,3">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_15" stretch="3,0,0,1,1">
        <item>
         <widget class="QLineEdit" name="searchlineedit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QComboBox" name="Sortcombobox">
          <item>
           <property name="text">
            <string>Sort by title</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Sort by artist</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="Addsongbutton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>5</horstretch>
            <verstretch>5</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Add Song</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_16" stretch="2,6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QListWidget" name="listWidget"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,3">
          <item>
           <widget class="QTableWidget" name="tableWidget"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Your Song History</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="listWidget_History"/>
            </item>
            <item>
             <widget class="QPushButton" name="clearhistorybutton">
              <property name="text">
               <string>Clear History</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_12" stretch="3,3,3">
        <item>
         <widget class="QLabel" name="currentsonglabel">
          <property name="text">
           <string>No Songs Playing</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_17">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="backHistoryButton">
            <property name="text">
             <string>Back history</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="prevbutton">
            <property name="text">
             <string>Prev</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="playbutton">
            <property name="text">
             <string>Play</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="nextbutton">
            <property name="text">
             <string>Next</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="favButton">
            <property name="text">
             <string>â™¡</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="deleteSong">
            <property name="text">
             <string>Delete Song</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
